src/demo.rs

```diff
@@ -689,15 +689,36 @@ fn handle_packet(packet: &Packet, state: &GameState) -> Vec<DemoMessage> {
 
 pub struct PrintVotes {
     votes: HashMap<u32, Vec<String>>,
+    shunted_messages: Vec<(u32, DemoMessage)>,
 }
 
 impl PrintVotes {
     #[must_use]
     pub fn new() -> Self {
         Self {
             votes: HashMap::new(),
+            shunted_messages: vec![],
         }
     }
+
+    fn process_vote_cast_event(
```

Comment:
I think the name could be slightly more informative of what it actually does, as `process` is quite vague. Maybe something along the lines of `get_vote_cast_message`?

Reply:
ye :+1: